scripts/install.in

```diff
@@ -76,14 +76,14 @@ fi
 
 tarball=$tmpDir/nix-@nixVersion@-$system.tar.xz
 
-require_util curl "download the binary tarball"
+require_util wget "download the binary tarball"
```

Comment:
Is macOS not shipping `curl` instead of `wget`?
How about just checking for both?

Reply:
Ah! That's right.
I've pushed a few changes to support both wget and curl. I'm not a pro in bash, so there's probably room for improvement, let me know what you think 🙂